src/test/java/sorinfoca/Tests/Tema8/pages/RefreshPage.java

```diff
@@ -0,0 +1,33 @@
+package sorinfoca.Tests.Tema8.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class RefreshPage {
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    public RefreshPage(WebDriver driver) {
+        this.driver = driver;
+        this.wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+    }
+
+    public void goToPage() {
+        driver.get("https://testpages.herokuapp.com/styled/refresh");
+    }
+
+    public boolean isIdTimestamp() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("refreshdate")));
+        return driver.findElement(By.id("refreshdate")).isDisplayed();
```

Comment:
This element is displayed even before refreshing the page so this condition will always be true. Try to make a method that returns instead the numerical value and check the value after refreshing the page with the one before refreshing the page. Thanks!